integrationTests/testInitializer.go

```diff
@@ -261,21 +269,36 @@ func CreateIeleVMAndBlockchainHook(accnts state.AccountsAdapter) (vmcommon.VMExe
 	return vm, blockChainHook
 }
 
-// CreateAddresFromAddrBytes creates a n address container object from address bytes provided
-func CreateAddresFromAddrBytes(addressBytes []byte) state.AddressContainer {
+// CreateAddressFromAddrBytes creates an address container object from address bytes provided
+func CreateAddressFromAddrBytes(addressBytes []byte) state.AddressContainer {
 	addr, _ := TestAddressConverter.CreateAddressFromPublicKeyBytes(addressBytes)
+	return addr
+}
 
+// CreateAddressFromHex creates an address container object from hex
+func CreateAddressFromHex() state.AddressContainer {
```

Comment:
I would call this CreateRandomAddress, as you are not providing the hex string yourself, so it can be anything.

Reply:
Changed.